integration/spark2/src/main/scala/org/apache/spark/sql/CarbonDatasourceHadoopRelation.scala

```diff
@@ -56,8 +56,8 @@ case class CarbonDatasourceHadoopRelation(
     paths.head,
     CarbonEnv.getDatabaseName(caseInsensitiveMap.get("dbname"))(sparkSession),
     caseInsensitiveMap("tablename"))
-  lazy val databaseName: String = carbonTable.getDatabaseName
-  lazy val tableName: String = carbonTable.getTableName
+  lazy val databaseName: String = identifier.getDatabaseName
+  lazy val tableName: String = identifier.getTableName
```

Comment:
I don't think it is necessary to take as lazy val, just access `identifier` in `toString` method